website/docs/reference/resource-properties/description.md

```diff
@@ -400,3 +476,68 @@ models:
 
 If mixing images and text, also consider using a docs block.
 
+### Add a description to a data test
+
+You can add a `description` property to a generic or singular data test. 
+
+#### Generic data test
+
+This example shows a generic data test that checks for unique values in a column for the `orders` model.
+
+<File name='models/<filename>.yml'>
+
+```yaml
+version: 2
+
+models:
+  - name: orders
+    columns:
+      - name: order_id
+        tests:
+            - unique
+              description: "The order_id is unique for every row in the orders model"
+```
+</File>
+
+#### Singular data test
+
+This example shows a singular data test that checks to ensure all values in the `payments` model are not negative (≥ 0).
+
+<File name='test/<filename>.yml'>
```

Comment:
```suggestion
<File name='tests/<filename>.yml'>
```
Below, the note referes to "tests" directory, so this should probably [match that](https://github.com/dbt-labs/docs.getdbt.com/pull/6733/files#diff-18a9904553cd02dbcfde65fb46c3198bcfc27fddf2598003be2e5eaff2c0d43aR519).